xblock/test/test_mixins.py

```diff
@@ -215,36 +218,43 @@ def has_support(self, view, functionality):
             )
 
 
+@ddt.ddt
 class TestXmlSerializationMixin(TestCase):
     """ Tests for XmlSerialization Mixin """
 
-    etree_node_tag = 'test_xblock'
-
     # pylint:disable=invalid-name
     class TestXBlock(XBlock):
         """ XBlock for XML export test """
+        etree_node_tag = 'test_xblock'
+
         field = String()
         simple_default = String(default="default")
-        simple_default_with_force_export = String(default="default", force_export=True)
+        force_export = String(default="default", force_export=True)
```

Comment:
To match the naming pattern for the next two vars, could you call this `simple_force_export`?